webapp/app/js/compareperf.js

```diff
@@ -104,7 +105,7 @@ comparePerf.controller('CompareCtrl', [ '$state', '$stateParams', '$scope', '$ro
                                                  $stateParams.e10s);
 
             if (seriesSummary != null && seriesSummary.signature !== undefined) {
-              seriesList.push(seriesSummary);
+              rawResultsMap[seriesSummary.signature] = {name: seriesSummary.name, platform: seriesSummary.platform};
```

Comment:
Could you break out the dictionary onto a new line like:
rawResultsMap[seriesSummary.signature] = {
   name: seriesSummary.name,
   platform: seriesSummary.platform
};